eno-core/src/main/java/fr/insee/eno/core/processing/out/LunaticProcessing.java

```diff
@@ -28,6 +30,7 @@ public void applyProcessing(Questionnaire lunaticQuestionnaire, EnoQuestionnaire
         EnoIndex enoIndex = enoQuestionnaire.getIndex();
         assert enoIndex != null;
         EnoCatalog enoCatalog = new EnoCatalog(enoQuestionnaire);
+        ShapefromAttributeRetrieval shapefromAttributeRetrieval = new ShapefromAttributeRetrievalFromVariableGroups();
```

Comment:
There's something that disturbs me here: could this instance be moved inside the Lunatic "shape from" processing?

Reply:
No, 

For the shape from processing to work, we need to link a calculated variable to the associated question. We cannot do that for a lunatic filter result/pairwise calculated variable in the shape from processing (or maybe by introducing a new attribute to the calculated variable, but i don't like this idea much). This information is only available when generating the filter result calculated variable in the filter result processing. Hence extracting the shape from attribute retrieval outside the processing.